extensions.json

```diff
@@ -94,6 +94,12 @@
       "id": "asabil.meson",
       "repository": "https://github.com/asabil/vscode-meson"
     },
+    {
+      "id": "asciidoctor.asciidctor-vscode",
```

Comment:
Typo: `asciidoctor-vscode`

Reply:
Commited requested changes. Hopefully this time without introducing a new typo...